### PR TITLE
test(e2e,playwright): 404 page

### PR DESCRIPTION
**Pull request by shootermv**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Related to #51705

### CODE REVIEW

**Comment by shootermv** [comment]
camperbot cant code

File: e2e/404.spec.ts [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+test.beforeAll(async ({ browser }) => {
+  page = await browser.newPage();
+  await page.goto('/404');
+});
+test.afterAll(async () => {
+  await page.close();
+});
+
+test('Should have page title "Page not found"', async () => {
+  await expect(page).toHaveTitle('Page not found| freeCodeCamp');
```
Comment (Tauseef-Hilal) [inline e2e/404.spec.ts:12]: You probably need to put a space before `|`

**Comment by samiralibabic** [comment]
There should be one spec for `404.ts` page and separate test for FourOhFour component `../components/FourOhFour/index.tsx`.

**Review by naomi-lgbt — APPROVED** [review]

**Comment by shootermv** [comment]
i closed this PR because i saw many other people did this too, and some did even better

**Pull request closed without merging**